components/Header/HeaderDashboard/index.module.scss

```
@@ -0,0 +1,155 @@
+@use '@/styles/index.scss' as *;
+
+.header-dashboard {
+  @include Flex(row, flex-end, center);
+  flex-grow: 1;
+
+  width: 100%;
+  @include ResponsiveHeight(70px, 70px, 60px);
+  @include ResponsivePadding(0 80px 0 40px, 0 32px 0 40px, 0 8px 0 16px);
+  border-bottom: 1px solid $gray-d9;
+
+  background: $white;
+  color: $black;
+
+  box-sizing: border-box;
+}
+
+.header-dashboard-container {
+  @include Flex(row, flex-end, center);
+  width: 100%;
+  height: 40px;

```

```suggestion
@include Size(100%, 40px);
```
size 믹스인에 %도 가능해서 이러면 좋을거같아용 ㅎㅎ